app/controllers/static_pages_controller.rb

```diff
@@ -18,7 +18,10 @@ def terms_conditions
   end
 
   def maintenance
```

Comment:
Now that we're no longer handling maintenance mode in the router, this action method is redundant as we only ever render maintenance mode from the `check_for_maintenance_mode` before action.

Reply:
Good point! Fixed.